tests.py

```diff
@@ -45,12 +45,12 @@ def test_app():
         "236",
         "232",
         "213",
-        "216",
         "212",
+        "216"
     ]
-    assert actual_value == expected_value, "Test case bla bla failed on empty pin"
-
+    assert actual_value == sorted(expected_value), "Test case failed on pin = 123"
```

Comment:
We should state our error message in such way we can change the values in the future. How about we completely remove the explanatory message, for now.